doc/tasks/phplint.md

```diff
@@ -0,0 +1,35 @@
+# PHPLint
+
+The PHPLint task will check your source files for syntax errors.
+
+```yaml
+# grumphp.yml
+parameters:
+    tasks:
+        phplint:
+            to_check: [ '.' ]
+            exclude: []
+            jobs: ~
+```
+
+**to_check**
+
+*Default: project root*
```

Comment:
In the other tasks, the default config is pasted here. You can clearify that it is loading the whole project in the text below.